src/components/Button.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from 'styled-components';
+
+const Btn = styled.button`
+  border-radius: 4px;
+  border: none;
+  background-color: #fdb755;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 14px;
+  letter-spacing: 0.5px;
+  cursor: pointer;
+  padding: ${(props) => props.padding};
+`;
+
+function Button({ text, padding }) {
+  return (
+    <>
+      <Btn type="submit" padding={padding}>
```

Comment:
Instead of setting padding via a prop you could accept a `className` prop and set it to `Btn`. This would look as follows

```
function Button({ text, className }) {
  return (
    <Btn type="submit" className={className}>
      {text}
    </Btn>
  );
}
```

You would customize the padding like this

```
const ButtonWithPadding = styled(Button)`
  padding: 12px;
`
```

Reply:
If you decide to go that way you can also replace the `text` prop with `children`. Then you don't even need the `Button` component but can export the styled component directly.